CMakeLists.txt

```diff
@@ -637,6 +637,7 @@ elseif(APPLE)
       MACOSX_BUNDLE TRUE
       MACOSX_RPATH TRUE
       BUILD_WITH_INSTALL_RPATH TRUE
+      INSTALL_NAME_DIR "@rpath"
```

Comment:
The `install_name` is only used by other applications that are trying to link LÖVR as a shared library, right?  Having a hard time understanding.

Reply:
I have written Mac apps for two decades and I'm still confused by install_name... Yes, I think it only applies to libraries. I see you took it out in latest dev; I'll play around with what's needed once I have this branch up and running on Linux and Windows.

Reply:
okay it works fine without this addition so I'm leaving it out...